### PR TITLE
Docs for Midea ccm15 new integration

### PR DESCRIPTION
**Pull request by ocalvo**
## Proposed change

Docs for Midea ccm15 new integration

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/94824
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/4495
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/ccm15.markdown [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  - '@ocalvo'
+ha_domain: ccm15
+ha_quality_scale: platinum
+ha_dhcp: true
+ha_platforms:
+  - climate
+ha_integration_type: integration
+---
+
+The `ccm15` integration allows you to integrate [Midea CCM15](https://mbt.midea.com/hvac-goods/midea-products-category/vrfs/vrf-controller/central-controller-ccm-15) devices in Home Assistant.
```
Comment (c0ffeeca7) [inline source/_integrations/ccm15.markdown]:
```suggestion
The CCM15 integration allows you to integrate [Midea CCM15](https://mbt.midea.com/hvac-goods/midea-products-category/vrfs/vrf-controller/central-controller-ccm-15) devices in Home Assistant.
```

**Review by c0ffeeca7 — DISMISSED** [review]
LGTM. Thank you, @ocalvo :+1:

File: source/_integrations/ccm15.markdown [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+Adding a CCM15 data converter to your Home Assistant instance can be done via the user interface, by using this My Button:
+{% my config_flow_start badge domain=page.ha_domain %}
+
+{% details "Manual configuration steps" %}
+
+1. Browse to your Home Assistant instance.
+1. Go to **{% my integrations title="Settings > Devices & Services" %}**.
+1. In the bottom right corner, select the
+  **{% my config_flow_start icon domain=page.ha_domain %}** button.
+1. From the list, select **CCM15** and follow the instructions on screen.
```
Comment (c0ffeeca7) [inline source/_integrations/ccm15.markdown]:
```suggestion
1. Browse to your Home Assistant instance.
2. Go to **{% my integrations title="Settings > Devices & Services" %}**.
3. In the bottom right corner, select the
  **{% my config_flow_start icon domain=page.ha_domain %}** button.
4. From the list, select **CCM15** and follow the instructions on screen.
```

**Review by c0ffeeca7 — APPROVED** [review]

File: source/_integrations/ccm15.markdown [hunk 3]
```diff
@@ -0,0 +1,43 @@
+---
+title: Midea CCM15 Data Converter
+description: Instructions on how to integrate a Midea CCM15 module into Home Assistant.
+ha_category:
+  - Climate
+ha_iot_class: Local Polling
+ha_release: 2023.07.0
```
Comment (frenck) [inline source/_integrations/ccm15.markdown:7]: @c0ffeeca7 You forgot to bump the HA release before merging this PR...

File: source/_integrations/ccm15.markdown [hunk 4]
```diff
@@ -0,0 +1,43 @@
+---
+title: Midea CCM15 Data Converter
```
Comment (frenck) [inline source/_integrations/ccm15.markdown:2]: This title differs from the PR upstream? Which one is correct @ocalvo?

File: source/_integrations/ccm15.markdown [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+{% details "Manual configuration steps" %}
+
+1. Browse to your Home Assistant instance.
+2. Go to **{% my integrations title="Settings > Devices & Services" %}**.
+3. In the bottom right corner, select the
+  **{% my config_flow_start icon domain=page.ha_domain %}** button.
+4. From the list, select **CCM15** and follow the instructions on screen.
+
+{% enddetails %}
```
Comment (frenck) [inline source/_integrations/ccm15.markdown:39]: This is not correct, we have an template for this which should have been included.

Please adjust this in a follow up PR @ocalvo.

/CC @c0ffeeca7

Comment (c0ffeeca7) [reply source/_integrations/ccm15.markdown:39]: Let me fix this, since I'm the one who merged this.
#30525

**Pull request merged**